.github/workflows/tests.yml

```diff
@@ -19,15 +20,7 @@ jobs:
           node-version: 14.x
 
       - name: Install dependencies
-        run: npm ci
-
-      - name: Install dependencies (in ./tests/integration/)
-        run: npm ci
-        working-directory: ./tests/integration/
-
-      - name: Link current version of plugin
-        run: npm link ../../
```

Comment:
I'm wondering if linking the package like this has some advantages over importing the reporter directly in the test. Seems like it could be a better setup for an integration test. It would ensure the package is exporting the correct files, verify the functional bits of `package.json`.

Reply:
Resolved by running the int tests in isolation (rather than starting the tests from root).